src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -138,8 +138,12 @@ public function get_grid_days( $date = null, $force = false ) {
 			$start = tribe_beginning_of_day( $day->format( Dates::DBDATETIMEFORMAT ) );
 			$end   = tribe_end_of_day( $day->format( Dates::DBDATETIMEFORMAT ) );
 
+			/*
+			 * We want events overlapping the current day, by more than 1 second.
+			 * This prevents events ending on the cutoff from showing up here.
+			 */
 			$day_query = tribe_events()->by_args( $repository_args )
-			                           ->where( 'date_overlaps', $start, $end )
+			                           ->where( 'date_overlaps', $start, $end, null, 2 )
```

Comment:
* :warning: Found precision alignment of 3 spaces.
* :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 8c8fd07bfecfa75886ea57382fe9dcc1</sub>